### ISSUE TITLE
MRQ preset doesn't load properly

### ISSUE THREAD

**Issue by softyoda** [Voulz/ArchVisTools#2]
Hi, I often use your add-on and really enjoy it, however, the Movie Render Queue load of sequence always reset the [ArchVis] Deferred Rendering Override Near Clip Plane to None (instead of -1) breaking the shift of the camera in the render.

Would it be possible to fix the default value when loading sequences ? It is time-consuming to always check this value before starting a render, worst if you have around 25 clips to render. Thanks for your help.

**Comment by Voulz** [comment]
Hello! I am actually working on a fix for that. Could you confirm which
version of UE you are using?

**Comment by softyoda** [comment]
I'm using the _4.27.2-18319896+++UE4+Release-4.2_ build.

**Comment by Voulz** [comment]
All good thanks.
I will have a beta update fixing this in the coming days/end of the week, do you mind sending me an email at voulz.git@gmail.com for you to try the beta when ready and let me know if it works properly?

**Comment by softyoda** [comment]
Alright, mail sent, thanks !

**Comment by Voulz** [comment]
This has been fixed in the latest update 1.4.1 of the plugin which is now available for update.
